Forms/MainForm.cs

```diff
@@ -1117,5 +1302,55 @@ private void TogglePatchSettings(bool v)
                 tPatch.Text = null;
             }
         }
+
+
+
+        private void WritePreset(string filename)
```

Comment:
You mentioned that the config files handle user logic. Where is that handled? My expectation is that it should just save the filename for it, since the user might want to frequently update their logic.